components/brave_shields/browser/ad_block_engine.cc

```diff
@@ -286,13 +307,30 @@ void AdBlockEngine::OnListSourceLoaded(const DATFileDataBuffer& filters,
 
 void AdBlockEngine::OnDATLoaded(const DATFileDataBuffer& dat_buf,
                                 const std::string& resources_json) {
+  DCHECK_CALLED_ON_VALID_SEQUENCE(sequence_checker_);
```

Comment:
That will prevent us from ocasionally calling deserializing from UI thread.

Reply:
the `OnDATLoaded` method was actually removed in @antonok-edm's PR. I don't think we load dat files anymore